### PR TITLE
console: make top navigation bar responsive (close #3680)

### LINKED ISSUE #3680: UI bug: top nav: where did the gear and (?) icons go?
v1.0.0

Today (as usual) I had a browser open to approximately 1000 pixels and thought I was losing my mind because the gear and ? in the top right were missing.

image

But no, it's just that the CSS needs some adjusting so they don't fall off the right side of the page.  (see attached image)

I almost NEVER use a browser set to max so anytime I need the hasura settings I need to resize my browser! 

Thanks!

### PR DESCRIPTION
**Pull request by abhi40308**
Earlier, when resizing the window, the gear and help icons would go out of window, this pull request fixes that issue.

### Description
fix #3680 

### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [ ] Server
- [x] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

### Solution and Design
New Behavior: 
ezgif com-gif-maker

### Steps to test and verify
Try resizing the window and check header navbar.

### Limitations, known bugs & workarounds

### CODE REVIEW

**Review by kolharsam — APPROVED** [review]
Looks great! 👌

**Review by beerose — CHANGES_REQUESTED** [review]
Instead of hiding `secure section`, please do the same as with another sidebar items — show icon, hide text.

**Review by beerose — APPROVED** [review]
Console

File: console/src/components/Main/Main.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ class Main extends React.Component {
                 tooltipPlacement={'left'}
                 customStyle={styles.secureSectionSymbol}
               />
-              &nbsp;Secure your endpoint
+              <span className={styles.secureSectionText}>&nbsp;Secure your endpoint</span>
```
Comment (beerose) [inline console/src/components/Main/Main.js:215]:
```suggestion
              <span className={styles.secureSectionText}>
                &nbsp;Secure your endpoint
              </span>
```

File: console/src/components/Main/Main.scss [hunk 2]
```diff
@@ -1531,6 +1532,34 @@
...
+  .secureSectionText {
+    display: none;
+  }
+
+  .shareSection {
+    display: none;
+  }
+}
+
+@media (max-width: 1000px) {
```
Comment (beerose) [inline console/src/components/Main/Main.scss:1545]:
```suggestion
@media (max-width: 1050px) {
```

**Pull request merged**